packages/netlify-cms-core/src/components/Editor/EditorToolbar.js

```diff
@@ -196,6 +196,10 @@ const SaveButton = styled(ToolbarButton)`
   ${buttons.lightBlue};
 `;
 
+const SaveButtonDisabled = styled(ToolbarButton)`
```

Comment:
How about adding:
```
  &[disabled] {
    ${buttons.disabled};
  }
```

to `SaveButton` and then passing `disabled={!hasChanged}` as prop.

TLDR: moving the condition to the CSS.

Reply:
I have changed this in my latest commits.
I also changed the cursor when hovering over a disabled button to the default instead of the pointer. This makes it more obvious that pressing the button won't do anything.